docs/configuration.rst

```diff
@@ -989,7 +989,7 @@ FSx
 Configuration for an attached FSx Lustre file system. See `FSx CreateFileSystem
 <https://docs.aws.amazon.com/fsx/latest/APIReference/API_CreateFileSystem.html>`_ for more information.
 
-FSx Lustre is only supported when ``base_os = centos7``.
+FSx Lustre is supported when ``base_os = centos7 | alinux``.
```

Comment:
do we mention anywhere that FSx is not supported with awsbatch as scheduler?

Reply:
no and no

Reply:
Ok, added. Here's what it looks like:

```
Beginning cluster creation for cluster: test
ERROR: FSx isn't supported with awsbatch as the scheduler, please submit a feature requestif you need it: https://github.com/aws/aws-parallelcluster/issues/new
```